modules/extension/xsd/xsd-sld/src/test/java/org/geotools/sld/v1_1/SLDExampleTest.java

```diff
@@ -192,6 +192,11 @@ public void testParseValidateVendorOptions() throws Exception {
         
         // check it passes validation
         List errors = validate(file);
+        if(errors.size()>0) {
+            for(int i=0;i<errors.size();i++) {
+                System.out.println(errors.get(i));
+            }
```

Comment:
Is it really needed? (Wondering if it has been added for debugging/testing purposes and forgotten)